app/javascript/logs/components/data_renderers/TextLog.vue

```diff
@@ -104,9 +104,13 @@ table.text-log-table {
       display: inline-block;
 
       &:not(:first-of-type) {
-        margin-top: 10px;
+        margin-top: 16px;
       }
     }
   }
+
+  td:has(ol, li) {
+    white-space: unset;
+  }
```

Comment:
This is sort of a compromise solution.

Above, we have `white-space: pre-wrap;` for `td`s. However, this looks terrible if/when there is a Markdown-parsed `ul` or `ol`. Therefore, here, we `unset` the `white-space` property at least for `td`s that have an `ol` and/or an `li`.